client/state/stats/lists/test/selectors.js

```diff
@@ -90,9 +91,12 @@ describe( 'selectors', () => {
 							2916284: {
 								statsStreak: {
 									'[["endDate","2016-06-01"],["startDate","2015-06-01"]]': {
-										1461961382: 1,
-										1464110402: 1,
-										1464110448: 1
+										streak: {},
```

Comment:
I don't see it here, but what is `streak` used for? Do we need it? Can the reducer simple track `data` so we can eliminate some of the conditions introduced in the selectors above?